src/systemcmds/pwm/pwm.c

```diff
@@ -649,18 +671,26 @@ pwm_main(int argc, char *argv[])
 						}
 					}
 
-					warnx("User abort\n");
+					PX4_INFO("User abort\n");
 					return 0;
 				}
 			}
 
-			usleep(2000);
+			/* Delay longer than the max Oneshot duration */
+
+			usleep(2542);
```

Comment:
Funny, what is the resolution of `usleep` on NuttX? Is it still only milliseconds? In which case it should probably be 3000.